Modules/_functoolsmodule.c

```diff
@@ -1073,6 +1073,7 @@ lru_cache_clear_list(lru_list_elem *link)
 static void
 lru_cache_dealloc(lru_cache_object *obj)
 {
+    _PyObject_GC_UNTRACK(obj);
     lru_list_elem *list = lru_cache_unlink_list(obj);
```

Comment:
Declaration after code is C99 syntax. The code will need rewriting for backporting to 2.7.